test/browser_tests/page_objects/page_newsroom-press-resources.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
+var stayInformedSection = require( '../shared_objects/stayInformedSection' );
+var rssSection = require( '../shared_objects/rssSection' );
+
+function PressResources() {
+
+  _assign( this, stayInformedSection, rssSection );
+
+  this.get = function() {
+    browser.get( '/newsroom/press-resources/' );
+  };
+
+  this.pageTitle = browser.getTitle;
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
+
+  this.subTitle = element( by.css( '.press-contacts_title' ) );
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.contactList = element( by.css( '.press-contacts_main-contact-list' ) );
+
+  this.contactListEmail = element.all(
+  by.css( '.press-contacts_main-contact-list .list_link' ) ).get( 0 );
+
+  this.contactListPhone = element.all(
+  by.css( '.press-contacts_main-contact-list .list_link' ) ).get( 1 );
+
+  this.contactPersons =
+  element.all( by.css( '.press-contacts_people .contact-person' ) );
+
+  this.pressSectionTitle = element( by.css( '.press-photos-bios_title' ) );
+
+  this.pressSectionIntro = element( by.css( '.press-section_intro' ) );
+
+  this.directorsBio =
+  element.all( by.css( '.press-photos-bios .contact-person' ) ).get( 0 );
+
+  this.directorsImage =
+  this.directorsBio.all( by.css( '.contact-person_photo' ) ).get( 0 );
+
+  this.directorsName =
+  this.directorsBio.all( by.css( '.contact-person_name' ) ).get( 0 );
+
+  this.directorsBioLink =
+  this.directorsBio.all( by.css( '.list__links .list__link' ) ).get( 0 );
```

Comment:
I'm not sure if these three things are worth testing. The markup for the bio and image links could easily change and there shouldn't be too many situation where these break and everything else on the page works.